### PR TITLE
Issue#260

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
issue #260
Hi [heronshoes](https://github.com/heronshoes),

I try to update on [readme.md](https://github.com/red-data-tools/red_amber/blob/main/README.md), and [readme.ja.md](https://github.com/red-data-tools/red_amber/blob/main/README.ja.md). But, I'm not sure these are the correct url, please check it out. Thanks!

### CODE REVIEW

File: README.ja.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![CI](https://github.com/heronshoes/red_amber/actions/workflows/ci.yml/badge.svg)](https://github.com/red-data-tools/red_amber/actions/workflows/ci.yml)
 [![Maintainability](https://api.codeclimate.com/v1/badges/b8a745047045d2f49daa/maintainability)](https://codeclimate.com/github/heronshoes/red_amber/maintainability)
 [![Test coverage](https://api.codeclimate.com/v1/badges/b8a745047045d2f49daa/test_coverage)](https://codeclimate.com/github/heronshoes/red_amber/test_coverage)
-[![Doc](https://img.shields.io/badge/docs-latest-blue)](https://heronshoes.github.io/red_amber/)
+[![Doc](https://img.shields.io/badge/docs-latest-blue)](https://github.com/red-data-tools/red_amber)
```
Comment (heronshoes) [inline README.ja.md:7]: It is simply OK to change `heronshoes` to `red-data-tools` 😄 

```suggestion
[![Doc](https://img.shields.io/badge/docs-latest-blue)](https://red-data-tools.github.io/red_amber/)
```

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 [![CI](https://github.com/heronshoes/red_amber/actions/workflows/ci.yml/badge.svg)](https://github.com/red-data-tools/red_amber/actions/workflows/ci.yml)
 [![Maintainability](https://api.codeclimate.com/v1/badges/b8a745047045d2f49daa/maintainability)](https://codeclimate.com/github/heronshoes/red_amber/maintainability)
 [![Test coverage](https://api.codeclimate.com/v1/badges/b8a745047045d2f49daa/test_coverage)](https://codeclimate.com/github/heronshoes/red_amber/test_coverage)
-[![Doc](https://img.shields.io/badge/docs-latest-blue)](https://heronshoes.github.io/red_amber/)
+[![Doc](https://img.shields.io/badge/docs-latest-blue)](https://github.com/red-data-tools/red_amber)
```
Comment (heronshoes) [inline README.md:7]:
```suggestion
[![Doc](https://img.shields.io/badge/docs-latest-blue)](https://red-data-tools.github.io/red_amber/)
```

**Review by heronshoes — APPROVED** [review]

**Comment by heronshoes** [comment]
Hi @viktoriussuwandi,
Thank you for the PR! Unfortunately links were not correct, but I have fixed them and will commit it. Thank you for your interest in the Issue in RedAmber!

**Comment by heronshoes** [comment]
Closes #260 .

**Comment by viktoriussuwandi** [comment]
Appreciate your corrections @heronshoes !

**Pull request merged**